webview-ui/package.json

```diff
@@ -68,7 +68,7 @@
 		"storybook": "^8.5.2",
 		"ts-jest": "^27.1.5",
 		"typescript": "^4.9.5",
-		"vite": "^6.0.5"
+		"vite": "5.3.3"
```

Comment:
Vite 6 changes the way HMR works, which breaks the example I was following. Will figure this out as a follow-up.